packages/atomic/src/pages/examples/commerce-website/homepage.html

```diff
@@ -12,10 +12,12 @@
       import {commerceEngine} from './engine.mjs';
 
       (async () => {
-        await customElements.whenDefined('atomic-commerce-interface');
-        const commerceInterface = document.querySelector('atomic-commerce-recommendation-interface');
+        await customElements.whenDefined('atomic-commerce-recommendation-interface');
+        const recommendationInterfaces = document.querySelectorAll('atomic-commerce-recommendation-interface');
 
-        await commerceInterface.initializeWithEngine(commerceEngine);
+        for (const recommendationInterface of recommendationInterfaces) {
+          await recommendationInterface.initializeWithEngine(commerceEngine);
+        }
```

Comment:
Nit:

```suggestion
        await Promise.all(recommendationInterfaces.map(interface => interface.initializeWithEngine(commerceEngine))
```

Reply:
It's a NodeList, so as discussed won't do (we'd have to use Array.from, to be able to call map)